### PR TITLE
Filter out irrelevant categories #1029

### PR DESCRIPTION
**Pull request by junkiattan**
#1029 

@nicolas-raoul @misaochan 

I have submitted a pull request for this issue.

Not sure whether this was intended. When I went through the set up instructions, my Android version was telling me to insert `formatted="false"` to some of the string xml files, in order for me to proceed with an error free build. I added them in in order to get my app running for this project. Not sure whether I did the right thing do ;/

Would be glad if you could advice

### CODE REVIEW

**Comment by nicolas-raoul** [comment]
Have a look here:
https://github.com/junkiattan/apps-android-commons/network

As you can see, our master has already changed a lot this since you forked.
That explains the conflicts.

The best thing for you to do is to rebase from our master.
Please let us know when you have done it.
Thanks! :-)

**Comment by junkiattan** [comment]
@nicolas-raoul 

I have done the rebase, however I realized that some of the files introduced or some of the updates is causing me to fail Travis even though I did not touch the mentioned files.

File: build.gradle [hunk 1]
```diff
@@ -7,7 +7,7 @@ buildscript {
         google()
     }
     dependencies {
-        classpath "com.android.tools.build:gradle:${project.gradleVersion}"
+        classpath 'com.android.tools.build:gradle:3.0.1'
```
Comment (misaochan) [inline build.gradle]: This should probably not be hardcoded. :)

**Comment by misaochan** [comment]
Are you still unable to build locally, or is the failure only in the Travis build? If master is failing Travis, PRs that are rebased on master fail Travis as well. In that case it is not a concern, we are usually okay with merging PRs that fail Travis for that reason. :)

The current build error is caused by a translatewiki problem, we should probably create a separate issue for it.

File: build.gradle (hunk 1 above)
Comment (junkiattan) [reply build.gradle]: that's strange. I did not touch that file. Ill reedit and rebuild my project again!

thanx @misaochan

**Comment by junkiattan** [comment]
@misaochan @nicolas-raoul 

after changing the class path again, I am still unable to build locally.

The error in the picture that I posted remains. I have done some online searching, and the best answer I could find was to remove the file, but since it was introduced in a previous merge I do not think that it is the best solution.

**Comment by misaochan** [comment]
Yeah, that was introduced by a translation commit. Sorry about that! This is tracked at #1135 .

In the meantime, AFAIK it is okay for you to make the changes needed to build locally, just don't commit them.

File: build.gradle (hunk 1 above)
Comment (misaochan) [reply build.gradle]: I believe Android Studio prompts you and asks if you would like to "update Gradle version", upon which it modifies the file. I generally set that to "Not ask again". ;)

**Comment by neslihanturan** [comment]
Hi @junkiattan , what should be result of this PR, can you describe? I have read the related issue, and expected not to see very old dates among categories. However, I am able to see them, am I missing something?

device-2018-02-15-153522

**Comment by junkiattan** [comment]
Hi @neslihanturan! 

What I did was just to skip all "in the XXXXs" categories except 2000s and 2010s as described in the issue. Maybe you can try typing in a specific key word like this [example](https://user-images.githubusercontent.com/99590/34404630-1fded974-ebf2-11e7-9651-0b360c1ed507.png).

**Comment by neslihanturan** [comment]
Oh so we are missing .XX format years. So I will merge this and hold the issue still open. Since there is extra work needed.

**Comment by junkiattan** [comment]
@neslihanturan 
I can help to add in the formatting of the years. But I would like to clarify about what is considered an old date for the year part. Referring to the picture you posted, is date with year 01 still considered old and to be filtered?

**Comment by neslihanturan** [comment]
Let us discuss it on the thread, since @nicolas-raoul suggested this idea, I think his user experience can guide us.

**Pull request merged**